From 5f5bcb6adb05a0cd88f8a67e09f4e4a9ff23edd3 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Wed, 1 Aug 2012 13:48:03 +0800
Subject: [PATCH 038/304] ENGR00215174 add fake display mode when HDMI read
 edid failed.

add fake display mode when HDMI read edid failed.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 common/display/display_mode_fb0.conf | 2 ++
 common/display/display_mode_fb2.conf | 2 ++
 common/display/display_mode_fb4.conf | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/common/display/display_mode_fb0.conf b/common/display/display_mode_fb0.conf
index ad0d324..cf41a80 100644
--- a/common/display/display_mode_fb0.conf
+++ b/common/display/display_mode_fb0.conf
@@ -1,3 +1,5 @@
+U:1920x1080p-60
+U:1280x720p-60
 S:1920x1080p-60
 S:1280x720p-60
 D:1024x768p-60
diff --git a/common/display/display_mode_fb2.conf b/common/display/display_mode_fb2.conf
index a2bf5f3..8dd147e 100644
--- a/common/display/display_mode_fb2.conf
+++ b/common/display/display_mode_fb2.conf
@@ -1,3 +1,5 @@
+U:1920x1080p-60
+U:1280x720p-60
 S:1920x1080p-60
 S:1280x720p-60
 D:1024x768p-60
diff --git a/common/display/display_mode_fb4.conf b/common/display/display_mode_fb4.conf
index a2bf5f3..8dd147e 100644
--- a/common/display/display_mode_fb4.conf
+++ b/common/display/display_mode_fb4.conf
@@ -1,3 +1,5 @@
+U:1920x1080p-60
+U:1280x720p-60
 S:1920x1080p-60
 S:1280x720p-60
 D:1024x768p-60
-- 
1.8.0

